File: app/src/main/java/io/tanjundang/study/knowledge/viewpager/PageThreeFragment.java
package io.tanjundang.study.knowledge.viewpager;


import android.os.Bundle;
import android.support.v4.app.Fragment;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

import io.tanjundang.study.R;

/**
 * A simple {@link Fragment} subclass.
 */
public class PageThreeFragment extends Fragment {


    public PageThreeFragment() {
        // Required empty public constructor
    }

    public static PageThreeFragment newInstance() {
        return new PageThreeFragment();
    }

    @Override
    public View onCreateView(LayoutInflater inflater, ViewGroup container,
                             Bundle savedInstanceState) {
        // Inflate the layout for this fragment
        return inflater.inflate(R.layout.fragment_page_three, container, false);
    }

}

File: app/src/main/java/io/tanjundang/study/knowledge/viewpager/PageThreeFragment.kt
package io.tanjundang.study.knowledge.viewpager


import android.os.Bundle
import android.support.v4.app.Fragment
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup

import io.tanjundang.study.R


class PageThreeFragment : Fragment() {

    override fun onCreateView(inflater: LayoutInflater?, container: ViewGroup?,
                              savedInstanceState: Bundle?): View? {
        return inflater!!.inflate(R.layout.fragment_page_three, container, false)
    }

    companion object {
        fun newInstance(): PageThreeFragment {
            return PageThreeFragment()
        }
    }

}
